ui/pages/confirmations/components/confirm/info/token-transfer/token-details-section.tsx

```diff
@@ -0,0 +1,79 @@
+import { TransactionMeta } from '@metamask/transaction-controller';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP } from '../../../../../../../shared/constants/network';
+import {
+  ConfirmInfoRow,
+  ConfirmInfoRowAddress,
+} from '../../../../../../components/app/confirm/info/row';
+import { ConfirmInfoSection } from '../../../../../../components/app/confirm/info/row/section';
+import {
+  AvatarNetwork,
+  AvatarNetworkSize,
+  Box,
+  Text,
+} from '../../../../../../components/component-library';
+import {
+  AlignItems,
+  BlockSize,
+  BorderColor,
+  Display,
+  FlexWrap,
+  TextColor,
+  TextVariant,
+} from '../../../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../../../hooks/useI18nContext';
+import {
+  getCurrentChainId,
+  getNetworkConfigurationsByChainId,
+} from '../../../../../../selectors';
+import { useConfirmContext } from '../../../../context/confirm';
+
+export const TokenDetailsSection = () => {
+  const t = useI18nContext();
+  const { currentConfirmation: transactionMeta } =
+    useConfirmContext<TransactionMeta>();
+
+  const chainId = useSelector(getCurrentChainId);
```

Comment:
We're actively removing these references now, so can we instead read this from the `currentConfirmation`?